src/components/booking/BookingSuccess.vue

```diff
@@ -0,0 +1,138 @@
+<script setup lang="ts">
+import { computed, watch } from "vue";
+import { storeToRefs } from "pinia";
+import { formattedDateAndTime } from "@/utils/formattedDate.js";
+import { useReservationStore } from "@/stores/reservation.js";
+import LoadingData from "@/components/global/LoadingData.vue";
+import ErrorMessage from "@/components/global/LoadingData.vue";
+import BookingSuccessTitle from "@/components/booking/BookingSuccessTitle.vue";
+
+import BaseButton from "../global/BaseButton.vue";
```

Comment:
We should stick to using the aliases everywhere 😄

```suggestion
import BaseButton from "@/components/global/BaseButton.vue";
```